packages/next/next-server/server/config.ts

```diff
@@ -66,11 +67,8 @@ const defaultConfig: { [key: string]: any } = {
 }
 
 const experimentalWarning = execOnce(() => {
-  console.warn(
-    chalk.yellow.bold('Warning: ') +
-      chalk.bold('You have enabled experimental feature(s).')
-  )
-  console.warn(
+  Log.warn(chalk.bold('You have enabled experimental feature(s).'))
```

Comment:
Intentionally dropped yellow?

Reply:
It's already prefixed by a yellow `warn  -`